### DISCUSSION TITLE
China leetcode website's url is different from international one.

### DISCUSSION THREAD

**Discussion by rhzxg** [lowlighter/metrics#1423] [Ideas]
I have been AKing algorithms problems on leetcode in China, the url of it is different from the international one. What if I want to add "plugin_leetcode" while using a Chinese account of leetcode?

**Comment by lowlighter** [comment]
Hi 👋 !

Currently the url is indeed hardcoded, but it shouldn't be too hard to implement if the endpoint are the same

Do you have an example user and the api endpoint for Chinese users so we can test it out ?

**Reply by rhzxg** [reply]
Of course! My pleasure! The international URL of leetcode is https://leetcode.com/ and the Chinese version of it is "https://leetcode.cn". As we can see, the only difference between them is the domain's suffix. For users, the URL should be "https://leetcode.cn/u/${User_Name}/" e.g. https://leetcode.cn/u/codingdogzxg/ is my profile link, please ignore my scores and code performance as I am a rookie in algorithms lol.

**Reply by rhzxg** [reply]
I've seen the endpoint API of Leetcode on GitHub: https://github.com/realVEct0r/leetcode-api, don't know if it can help.

**Reply by lowlighter** [reply]
I started looking into it but it might take a bit more time than expected
It's definitely doable, but the GraphQL API is slightly different on the chinese version so I need to map back the data to make it work 😅

**Reply by rhzxg** [reply]
Sure thing! I am already excited about it! Looking for your good news!
